cocos/core/renderer/scene/model.ts

```diff
@@ -271,7 +271,7 @@ export class Model {
     }
 
     /**
-     * @en enabled
+     * @en Whether the model is enabled
```

Comment:
I think the wording of "model is enabled" is inappropriate. Does it mean visible here?